packages/recipes/src/createRuntimeFn.ts

```diff
@@ -64,5 +64,10 @@ export const createRuntimeFn = <Variants extends VariantGroups>(
 
   runtimeFn.variants = () => Object.keys(config.variantClassNames);
 
+  runtimeFn.classNames = {
```

Comment:
To prevent doing work at runtime for all users, we could instead make this a JS getter?